test/parallel/test-tls-timeout-server-2.js

```diff
@@ -1,32 +1,32 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
 if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
-var tls = require('tls');
+const tls = require('tls');
 
-var fs = require('fs');
+const fs = require('fs');
 
-var options = {
+const options = {
   key: fs.readFileSync(common.fixturesDir + '/keys/agent1-key.pem'),
   cert: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem')
 };
 
-var server = tls.createServer(options, function(cleartext) {
-  var s = cleartext.setTimeout(50, function() {
+const server = tls.createServer(options, common.mustCall(function(cleartext) {
+  const s = cleartext.setTimeout(50, common.mustCall(function() {
     cleartext.destroy();
     server.close();
-  });
+  }));
   assert.ok(s instanceof tls.TLSSocket);
-});
+}));
 
-server.listen(0, function() {
+server.listen(0, common.mustCall(function() {
```

Comment:
Unsure what the convention of using `common.mustCall` is. This one could be removed and the test would still work as expected, since line 18 also has a `common.mustCall`.

My instinct is to use `common.mustCall` as much as possible to avoid any problems in the future as this code is further refactored, but happy to change this based on what other people tend to do.